base/access_provider/api.go

```diff
@@ -19,6 +19,14 @@ type AccessSyncConfig struct {
 	Prefix     string
 }
 
+// AccessSyncImportConfig contains all necessary configuration parameters to import Data from Raito into DS
+type AccessSyncImportConfig struct {
```

Comment:
I'm wondering if we shouldn't make these names (also the ones from the interface methods) much more clear.
Because import and export can be seen from the 2 directions.

Something like: AccessSyncToTarget and AccessSyncFromTarget